config.js

```diff
@@ -7,6 +7,12 @@ config.server = {
     lifetimeCheckInterval: 1000,        // Minimum interval between lifetime checks in ms
     udpWindow: 100,
     defaultType: 'Device',
+    types: [
```

Comment:
Each time a new configuration item is added to config.js we need in addition:

1. Describe it at https://github.com/telefonicaid/lwm2m-node-lib#-configuration
2. Define a env var variable for it (LWM2M_XXXX) and add it to the table of in the above section
3. Modify the code to process the env var as equivalent way of providing configuration

In this case I'm not sure about 2 and 3, as the value of this variable is an structured object, so it is not so simple. How the current code deais with similar case (i.e. config.formats)?

Reply:
I didn't add this variable and it is already described in the configuration section of the README.md file

Reply:
Ok, I understand you refer to this description in the README.md (https://github.com/telefonicaid/lwm2m-node-lib#-configuration)

> The southbound interfaces can be configured in the `server.types` configuration parameter. This parameter is a list of objects composed of two attributes:

So you mean you have not added any new configuration variable in config.js but using and existing one. Is my undersrtanding correct?

Reply:
Yes

Reply:
Thanks for the clarification!

NTC 

(NTC = Nothing To Change is another of the mark we use in pull request. In thi case, to tell that a given comments thread discussion is over)